pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -301,12 +293,6 @@ func (mi *StorageInitializerInjector) InjectStorageInitializer(pod *v1.Pod) erro
 	userContainer.VolumeMounts = append(userContainer.VolumeMounts, sharedVolumeReadMount)
 	if transformerContainer != nil {
 		transformerContainer.VolumeMounts = append(transformerContainer.VolumeMounts, sharedVolumeReadMount)
-		// Change the CustomSpecStorageUri env variable value to the default model path if present
-		for index, envVar := range transformerContainer.Env {
-			if envVar.Name == constants.CustomSpecStorageUriEnvVarKey && envVar.Value != "" {
-				transformerContainer.Env[index].Value = constants.DefaultModelLocalMountPath
-			}
-		}
```

Comment:
We still need to mount to the transformer but we only load the storage uri from kserve container.

Reply:
Discussed with siva this can be safely removed as we do not allow setting storage uri env on transformer-container now.